src/components/LiveFeedsSlide.jsx

```diff
@@ -45,49 +74,309 @@ const CloseButton = styled(Button)(({ theme }) => ({
 const LiveFeedsSlide = (props, ref  ) => {
 
   const { dispatch, state } = useContext(store);
-  const { hideLiveFeeds } = props
+  const [filters, setFilters] = useState({
+    my: true,
+    general: true
+  });
+  const { hidelivefeeds } = props
+  const queryClient = useQueryClient();
 
   const clearAll = () => {
-    dispatch(Actions.resetNotifications(true))
+    // dispatch(Actions.resetNotifications(true))
+    queryClient.setQueryData([QUERY_KEYS.liveFeeds, { filterKey: "MyAlerts" }], [])
```

Comment:
nit, If you want to clear all events, you can simply delete the query data instead of setting it as empty array